gpu_ocean/SWESimulators/CDKLM16.py

```diff
@@ -90,7 +96,11 @@ def __init__(self, \
         g: Gravitational accelleration (9.81 m/s^2)
         f: Coriolis parameter (1.2e-4 s^1), effectively as f = f + beta*y
         r: Bottom friction coefficient (2.4e-3 m/s)
+        subsample_f: Subsample the coriolis f when creating texture by factor (for beta plane)
         angle: Angle of rotation from North to y-axis as a texture (cuda.Array) or numpy array
+        subsample_angle: Subsample the angles given as input when creating texture by factor
+        latitude: Specify latitude. This will override any f and beta plane already set
```

Comment:
Based on the use of the latitude, it seems that latitude should be given in radians (see OceanographicUtilities.calcCoriolisParams). However, the latitude in NorKyst-800 is given in degrees.

I suggest we specify that it is given in degrees, and then call OceanographicUtilities.degToRad before filling the texture.

Reply:
Vi bruker radianer som input, se NorKyst800-notebook'en. Det er brukers ansvar å konvertere til radianer på utsiden.

Reply:
Dette matcher da 'angle' som er vinkel mot nord (som også er radianer)

Reply:
Den er grei. Men kan du da legge til i doc-stringen "latitude: Specify latitue in radians. This will..."? Sånn at det ikke er noe tvil.

Når vi lager noe som støtter interaksjon med NetCDF, så tenker jeg vi kan koste på oss en ekstra spesifisering når vi har en parameter som ikke er det samme som sin netcdf-ekvivalent.

Reply:
Allerede lagt til (for både lat og for angle)